app/views/aleph/_list_local_locations.html.erb

```diff
@@ -30,14 +30,14 @@
 
           <span class="availability-status">
             <i class="fa fa-check" aria-hidden="true"></i>
-            Available
+            <%= s[:label] %>
```

Comment:
I think with this changeset, the `<% if s[:available?] %>` condition can be removed as the `if/else` now prints the same thing.

Reply:
Sadly no; the ✔️ or ❌ is different depending on availability.

Reply:
Ha. I missed that. 👍